js/samples/advanced-csharp-multiple-resources/Views/Home/Index.cshtml

```diff
@@ -80,15 +74,8 @@
 
         function handleError(logMessage, error) {
             console.log(logMessage, error);
-
-            if (error && error.responseJSON) {
-                $("#error").text(error.responseJSON.message);
-            }
-        }
-
-        function decryptToken(encryptedToken) {
-            return atob(encryptedToken);
+            document.getElementById("error").textContent = "An error occurred while launching Immersive Reader.";
```

Comment:
is it safe to assume that you don't want to expose actual error

Reply:
makes sense, showing actual error now